unicode_smilies/unicode_smilies.php

```diff
@@ -1,260 +1,2042 @@
-//	Smilies::add($b, ':alpha', '&#x;');
-//	Smilies::add($b, ':marine', '&#x;');
-	Smilies::add($b, ':sabre', '&#x1F5E1;');
-//	Smilies::add($b, ':tank', '&#x;');
-//	Smilies::add($b, ':viking', '&#x;');
-//	Smilies::add($b, ':gangs', '&#x;');
-//	Smilies::add($b, ':dj', '&#x;');
-//	Smilies::add($b, ':elvis', '&#x;');
-	Smilies::add($b, ':violin', '&#x1F3BB;');
+Smilies::add($b, ':tea', '&#x2615;');
+Smilies::add($b, ':tee', '&#x2615;');
+//Smilies::add($b, ':drool', '&#x1F924;'); // Bad client support
+
+Smilies::add($b, ':cool', '&#x1F192;');
+//Smilies::add($b, ':vomit', '&#x1F92E;'); // Bad client support
+Smilies::add($b, ':golf', '&#x1F3CC;');
+Smilies::add($b, ':football', '&#x1F3C8;');
+Smilies::add($b, ':tennis', '&#x1F3BE;');
+//Smilies::add($b, ':alpha', '&#x;');
+//Smilies::add($b, ':marine', '&#x;');
+Smilies::add($b, ':sabre', '&#x1F5E1;');
+//Smilies::add($b, ':tank', '&#x;');
+//Smilies::add($b, ':viking', '&#x;');
+//Smilies::add($b, ':gangs', '&#x;');
+//Smilies::add($b, ':dj', '&#x;');
+//Smilies::add($b, ':elvis', '&#x;');
+Smilies::add($b, ':violin', '&#x1F3BB;');
+
+
+//Neu hinzugefügte Unicode Emoji von Matthias Ebers
+//face-smiling
+    Smilies::add($b, ':grinning face', '&#x1F600');
+    Smilies::add($b, ':grinning face with big eyes', '&#x1F603');
+    Smilies::add($b, ':grinning face with smiling eyes', '&#x1F604');
+    Smilies::add($b, ':beaming face with smiling eyes', '&#x1F601');
+    Smilies::add($b, ':grinning squinting face', '&#x1F606');
+    Smilies::add($b, ':laughing', '&#x1F606');
+    Smilies::add($b, ':grinning face with sweat', '&#x1F605');
+    Smilies::add($b, ':rolling on the floor laughing', '&#x1F923');
+    Smilies::add($b, ':face with tears of joy', '&#x1F602');
+    Smilies::add($b, ':slightly smiling face', '&#x1F642');
+    Smilies::add($b, ':upside-down face', '&#x1F643');
+    Smilies::add($b, ':winking face', '&#x1F609');
+    Smilies::add($b, ':smiling face with smiling eyes', '&#x1F60A');
+    Smilies::add($b, ':smiling face with halo', '&#x1F607');
+    Smilies::add($b, ':grinsendes gesicht', '&#x1F600');
+    Smilies::add($b, ':grinsendes gesicht mit großen augen', '&#x1F603');
+    Smilies::add($b, ':grinsendes gesicht mit lachendes augen', '&#x1F604');
+    Smilies::add($b, ':strahlendes gesicht mit lächelnden augen', '&#x1F601');
+    Smilies::add($b, ':grinsendes schiel-gesicht', '&#x1F606');
+    Smilies::add($b, ':lachen', '&#x1F606');
+    Smilies::add($b, ':grinsendes gesicht schweißgebadetes', '&#x1F605');
+    Smilies::add($b, ':sich vor Lachen auf dem boden wälzend', '&#x1F923');
+    Smilies::add($b, ':gesicht vor freudentränen weinen', '&#x1F602');
+    Smilies::add($b, ':leichtes lachen', '&#x1F642');
+    Smilies::add($b, ':verkehrtes gesicht', '&#x1F643');
+    Smilies::add($b, ':zwinkerndes gesicht', '&#x1F609');
+    Smilies::add($b, ':lachendes gesicht mit lächelnden augen', '&#x1F60A');
+    Smilies::add($b, ':lachendes gesicht mit heiligenschein', '&#x1F607');
```

Comment:
I'm unsure about these (few) German translations for the emoticons. Why those and not all, and why not every language that exists on Earth? ;-)

Reply:
They are removing